pkg/storage/inmemory/article.go

```diff
@@ -0,0 +1,97 @@
+package inmemory
+
+import (
+	"errors"
+	"github.com/reiver/go-porterstemmer"
+	"news-aggregator/pkg/model"
+	"strings"
+	"time"
+)
+
+// ArticleInMemory is a struct that contains the in-memory database for articles.
+type ArticleInMemory struct {
+	Articles []model.Article
+	nextID   int
+}
+
+func New() *ArticleInMemory {
+	return &ArticleInMemory{
+		Articles: []model.Article{},
+		nextID:   1,
```

Comment:
strage. why 1? why not 0? or 100 ?

Reply:
This is an initialization of inmemory storage, then we incrementing this value for articles id in DB